pkg/k8shandler/statefulset.go

```diff
@@ -34,9 +35,21 @@ type statefulSetNode struct {
 	client client.Client
 
 	esClient elasticsearch.Client
+
+	l logr.Logger
+}
+
+// L is the logger relative to this node
+//
+// TODO remove this construct when context.Context is passed and it should contain any relevant contextual values
+func (n *statefulSetNode) L() logr.Logger {
```

Comment:
can we unify with deployment.go on the `n` vs `node` vs `sn` ?

Reply:
I used n for now and the go lint/vet only says keep all of the same struct receivers the same name. There's no Go vet/lint setting that suggests making them the same across multiple structs, but I'm okay doing that in a separate PR if that's okay with you?